mods/ts/rules/civilian-infantry.yaml

```diff
@@ -270,13 +272,16 @@ VISC_LRG:
 		HP: 500
 	Armament:
 		Weapon: SlimeAttack
+		FireDelay: 10
```

Comment:
Uploaded replay to http://tdx.ppmsite.com/OpenRA-2017-01-15T184844Z.zip

I first sent a bike to a visceroid, next a buggy to a tib fiend, in both cases they actively attacked my vehicles.